### PR TITLE
GLT-672 Updating OICR sequencer details

### PR DESCRIPTION
**Pull request by tonydebat**

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration/V8012__sequencer_update.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+DELETE FROM SequencingParameters WHERE SequencingParameters.name = 'v4 1×136';
+
+DELETE FROM Platform WHERE Platform.name IN ('Solid','LS454','IonTorrent','OxfordNanopore');
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration/V8012__sequencer_update.sql]: Do we want to fully delete these before migration? We do have some historical Solid and IonTorrent data.

Comment (apmasell) [inline sqlstore/src/main/resources/db/migration/V8012__sequencer_update.sql]: Agreed. I thought the plan was to filter them based on installed sequencers in the UI, not the DB.

**Comment by tonydebat** [comment]
Platforms are no longer removed from the database. Platforms are hidden if there are no associated sequencers. This change is in the tgac/develop branch: https://github.com/TGAC/miso-lims/pull/306

**Comment by apmasell** [comment]
:+1:

**Comment by callunity** [comment]
:+1:

**Pull request merged**